service/src/main/java/tds/assessment/web/endpoints/AdminSubjectController.java

```diff
@@ -0,0 +1,46 @@
+package tds.assessment.web.endpoints;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import tds.assessment.SetOfAdminSubject;
+import tds.assessment.services.AdminSubjectService;
+import tds.assessment.web.resources.SetOfAdminSubjectResource;
+import tds.common.web.exceptions.NotFoundException;
+
+/**
+ * Controller handling admin subject related entities
+ */
+@RestController
+@RequestMapping("/assessments/admin-subject")
```

Comment:
I like the hyphen for the URL instead of camelCase so just calling that out because I think this is the first time I've seen it.  Not sure if there were other routes using camelCase or not, but I think we should make the all lowercase hyphenated the standard like this.  @jeffjohnson9046